### ISSUE TITLE
zur Diskussion: `<lb break="no"/>` einführen?

### ISSUE THREAD

**Issue by cthomasdta** [deutschestextarchiv/dtabf#66]
Es ist ja öfter so, dass die Trennstriche fehlen, nicht nur in Handschriften, sondern auch in älteren Drucken. In der ehd und auch anderswo wird das getaggt als `<lb break="no"/>`, z.B.:
`Besorgniß vor bös<lb break="no"/>artigen Fibern`

Sollte `<lb break="no"/>` auch im DTABf erlaubt werden?

Ich finde die Alternativen, wenn ich sie denn richtig sehe:
`Besorgniß vor bös<supplied>-</supplied><lb/>artigen Fibern`
oder 
`Besorgniß vor <choice><orig>bös</orig><reg>bös-</reg></choice><lb/>artigen Fibern`
nicht besser, eher schlechter.

**Comment by funkyfuture** [comment]
ich verstehe [die TEI guidelines](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.breaking.html) so, dass dieses attribut keine information über (fehlende) trennzeichen trägt. mit dem attributwert `no` wird ein nicht-whitespace auf orthographischer ebene ausgezeichnet.

das `supplied` tag wäre per definition als alternative nicht geeignet. die `choice`-variante kann für die behandlung gemutmaßter setzfehler u. dgl. genutzt werden.

um die ehd zu legalisieren wäre das also notwendig. ;-)

**Comment by cthomasdta** [comment]
@funkyfuture  Verstehe ich dich richtig, dass du das hier https://edition-humboldt.de/richtlinien/frameworks/basis/text/zeilenumbruch beschriebene Vorgehen falsch findest?

**Comment by textloop** [comment]
Als kleine Anmerkung von mir dazu: 

So wir ihr es bei jetzt bei der **edition humboldt digital** macht, hat es sich auch für mich als bester Kompromiss für sehr viele Projekte herauskristallisiert.

**Comment by StefanDumont** [comment]
@cthomasdta Hmm... in deinem Ticket sind irgendwie zwei Aspekte vermischt: Erlauben von `@break="no"` in DTABf (finde ich sinnvoll) um anzuzeigen, dass der Zeilenumbruch mitten in einem Wort geschieht und der Umgang mit fehlenden Trennzeichen. Beides hat erstmal nichts miteinander zu tun. In der ehd wird bisher auch nicht ein irrtümlich fehlendes Trennzeichen kodiert (wäre mir jetzt neu), sondern nur die vorhandenen speziell getaggt, um sie in der Leseansicht auszublenden.

**Comment by cthomasdta** [comment]
Nein, das ist ein Missverständnis. Die beiden schlecheren Alternativen mit `<supplied>` oder `choice|orig|reg`, die ich angegeben habe, sind die Workarounds, die wir im DTA manchmal haben. Dort muss, eben solange wir `<lb break="no"/>` nicht haben, der Bindestrich auf die eine oder die andere Weise ergänzt werden, damit die Verarbeitung (inkl. Tokenisierung und Suchindizes) funktioniert.

Daher mein Vorschlag,  `<lb break="no"/>` *ohne* Supplieren oder Regularisieren des fehlenden Bindestrichs einzuführen.

**Comment by StefanDumont** [comment]
Ah, okay, das macht Sinn. Hatte das jetzt missverstanden.

**Comment by funkyfuture** [comment]
> @funkyfuture Verstehe ich dich richtig, dass du das hier https://edition-humboldt.de/richtlinien/frameworks/basis/text/zeilenumbruch beschriebene Vorgehen falsch findest?

nein, ich wollte vor allem auf den punkt aufmerksam machen, dass bei den phänomenen keine trennzeichen fehlen, und deshalb die erwähnten workarounds semantisch falsch wären, weil sie solche hinzufügten. die ehd macht das also ganz richtig und das dtabf müsste entsprechend erweitert werden, wenn denn derselbe standard verwendet werden soll.

**Comment by burki** [comment]
Ich habe kein Problem with dem break="no" Attribut.

Ich lese die Guidelines aber wie @funkyfuture und @StefanDumont  so, dass man mit 
  Um &lt;lb break="no" />bruch
markieren kann, dass "Umbruch" und nicht "Um" und "bruch" im Text steht; allerdings wäre es genau so korrekt, Um-&lt;lb break="no" />bruch zu setzen (vgl. das Example ne-&lt;lb break="no"/>ver https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-lb.html) in den Guidelines.

Das Problem das ich sehe, ist dass wir uns also eine Zweideutigkeit bei der Kodierung einhandeln: 
  Ist Um-&lt;lb break="no" />bruch identisch mit Um-&lt;lb />bruch ?
Und falls ja, welches ist die bevorzugte Variante? (präziser wäre die mit break="no", aber das wäre eine Änderung zur bestehenden Kodierung).

**Comment by cthomasdta** [comment]
Hier noch ein Beispiel aus dem DTA selbst, weshalb wir das brauchen:
http://www.deutschestextarchiv.de/nn_fusspfad_1492 siehe z.B. http://www.deutschestextarchiv.de/nn_fusspfad_1492/29: 
`weltli <lb/>chen`, `spri<lb/>cht`, `vn/ <lb/>moͤglich` usw.

⇒ auch die Verarbeitung / Indexierung im DTA müsste dann angepasst werden, sobald wir `break="no"` in `<lb/>` einführen.

**Comment by textloop** [comment]
Vielleicht hilft es, wenn man das mal in Richtung einer korrekten Ausgabe in ein anderes Format,  z.B. HTML, denkt. Gesetzt den Fall, dass die Zeilenumbrüche des Originals dort ignoriert werden sollen und der Text frei fließen soll.

> Ist Um-<lb break="no" />bruch identisch mit Um-<lb />bruch ?
Das ist nicht identisch, denn es können ja z.B. auch durch Bindestrich zusammengesetzte Wörter (wie etwa `Lotto-Annahmestelle`) und Auslassungen vorkommen (wie etwa Ein- und Ausgang).

Für eine Ausgabe nach HTML könnte 
`<lb/>` als Leerzeichen werden und `<lb break="no"/>` einfach entfallen. 

Ich nehme mal alle Beispiel von oben: 
`bös<lb break="no"/>artigen` würde als `bösartigen` ausgegeben werden
`Lotto-<lb break="no"/>Annahmestelle` würde zu  `Lotto-Annahmestelle`
`Ein-<lb/>und Ausgang` würde zu `Ein- und Ausgang`

`<supplied>-</supplied><lb/>` oder `<choice><orig>bös</orig><reg>bös-</reg></choice><lb/>` würde man dann nur setzen, wenn ein Bindestrich entweder durch den Autoren oder den Editoren ergänzt wurde, weil er im  Original fehlte.

**Comment by cthomasdta** [comment]
Noch eine Randbemerkung, auf die wir nicht unbedingt Rücksicht in der Erweiterung des DTABf nehmen müssen, aber AvH-unselbst (Bern) nutzt neben `<lb break="no">` auch die Werte `"yes"` und `"maybe"`, was wohl bei der Herstellung der Druckausgabe hilfreich für den Satzdienstleister war. Aber im DTABf reicht es m.E., `"no"` einzuführen.

**Comment by susannehaaf** [comment]
Beschluss DTABf-Steuerungsgruppe vom 7.1.2021:

- <lb break="no"/> wird eingeführt als Level-3-Element für Projekte, die eine Lesefassung ihrer Digitalisate anbieten möchten
- Wenn es in einem Text überhaupt eingesetzt wird, ist es dann konsequent einzusetzen (nicht sporadisch), um Ambiguitäten zu vermeiden
- für Trenn-/Bindestrich ist eine separate Strategie nötig: Hier wird auf die entsprechenden Unicode-Entitäten zurückgegriffen: 
  - U+002D (HYPHEN-MINUS) für "harten" Trennstrich (`Lotto-<lb break="no"/>Annahmestelle`)
  - U+2011 (NON-BREAKING HYPHEN) für "weichen" Trennstrich (`bös­‑<lb break="no"/>artigen`)
  - U+2013 (EN DASH) für Bis-Strich/Bindestrich (`2020–<lb break="no"/>2021`)
